backend/src/Service/Admin/Report/ReportService.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Service\Admin\Report;
+
+use App\AutoMapping;
+use App\Constant\Order\OrderStateConstant;
+use App\Constant\StoreOwner\StoreProfileConstant;
+use App\Response\Admin\Report\StatisticsForAdminGetResponse;
+use App\Service\Admin\Order\AdminOrderService;
+use App\Service\Admin\StoreOwner\AdminStoreOwnerService;
+
+class ReportService
+{
+    private AutoMapping $autoMapping;
+    private AdminStoreOwnerService $adminStoreOwnerService;
+    private AdminOrderService $adminOrderService;
+
+    public function __construct(AutoMapping $autoMapping, AdminStoreOwnerService $adminStoreOwnerService, AdminOrderService $adminOrderService)
+    {
+        $this->autoMapping = $autoMapping;
+        $this->adminStoreOwnerService = $adminStoreOwnerService;
+        $this->adminOrderService = $adminOrderService;
+    }
+
+    public function getStatisticsForAdmin(): StatisticsForAdminGetResponse
```

Comment:
You literally did not do anything related to my past comment, still 6000 queries. 
I will ask Habib @habibdevsy  to leave you a voice with some explaning.